tests/output/ConsoleInteractionTest.py

```diff
@@ -712,9 +712,7 @@ def test_show_bear_with_content(self):
                              '  Needed Settings:\n'
                              '   * setting1: Required Setting.\n\n'
                              '  Optional Settings:\n'
-                             '   * setting2: Optional Setting. ('
-                             "Optional, defaults to 'None'."
-                             ')\n\n'
+                             '   * setting2: Optional Setting.\n\n'
```

Comment:
```(Optional, defaults to 'whatever'.)``` was coming because we were doing like 
```doc_comment.param_dict.get(arg, cls.str_nodesc) + ' (' +
                     cls.str_optional.format(
                         defaults[i-num_non_defaults]) + ')'```

Since the issue asks for this concatenation to be removed... that ```(Optional, defaults to 'whatever')``` part will not be displayed... If UI should remain same, then we have to have this concatenation. This part in the UI is coming because of that concatenation.